spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/hystrix/HystrixCircuitBreakerConfiguration.java

```diff
@@ -70,6 +71,11 @@ public HasFeatures hystrixFeature() {
 		return HasFeatures.namedFeatures(new NamedFeature("Hystrix", HystrixCommandAspect.class));
 	}
 
+	@Bean
+	public HystrixConcurrencyStrategy hystrixConcurrencyStrategy(){
```

Comment:
Should this be opt-in (`@ConditionalOnProperty`)? Since there can only be one `HystrixConcurrencyStrategy`?